src/conduits/RwaOutputConduit3.sol

```diff
@@ -0,0 +1,380 @@
+    address public to;
+
+    /// @notice Whitelist for addresses which can be picked. `bud[who]`
+    mapping(address => uint256) public bud;
+    /// @notice Addresses with push access on this contract. `may[usr]`
+    mapping(address => uint256) public may;
+
+    /// @notice Exit address
+    address public quitTo;
+
+    /**
+     * @notice `usr` was granted admin access.
+     * @param usr The user address.
+     */
+    event Rely(address indexed usr);
+    /**
+     * @notice `usr` admin access was revoked.
+     * @param usr The user address.
+     */
+    event Deny(address indexed usr);
+    /**
+     * @notice `usr` was granted push access.
+     * @param usr The user address.
+     */
+    event Mate(address indexed usr);
+    /**
+     * @notice `usr` push access was revoked.
+     * @param usr The user address.
+     */
+    event Hate(address indexed usr);
+    /**
+     * @notice `usr` was granted operator access.
+     * @param usr The user address.
+     */
+    event Hope(address indexed usr);
+    /**
+     * @notice `usr` operator access was revoked.
+     * @param usr The user address.
+     */
+    event Nope(address indexed usr);
+    /**
+     * @notice `who` address whitelisted for pick.
+     * @param who The user address.
+     */
+    event Kiss(address indexed who);
+    /**
+     * @notice `who` address was removed from whitelist.
+     * @param who The user address.
+     */
+    event Diss(address indexed who);
+    /**
+     * @notice `who` address was picked as the recipient.
+     * @param who The user address.
+     */
+    event Pick(address indexed who);
+    /**
+     * @notice `wad` amount of Dai was pushed to the recipient `to`.
+     * @param to The Dai recipient address
+     * @param wad The amount of Dai
+     */
+    event Push(address indexed to, uint256 wad);
+    /**
+     * @notice A contract parameter was updated.
+     * @param what The changed parameter name. Currently the supported values are: "quitTo".
+     * @param data The new value of the parameter.
+     */
+    event File(bytes32 indexed what, address data);
+    /**
+     * @notice The conduit outstanding gem balance was flushed out to `quitTo` address.
+     * @param quitTo The quitTo address.
+     * @param wad The amount flushed out.
+     */
+    event Quit(address indexed quitTo, uint256 wad);
+
+    /**
+     * @notice Defines PSM and quitTo addresses and gives `msg.sender` admin access.
+     * @param _psm PSM contract address.
+     */
+    constructor(address _psm) public {
+        GemAbstract _gem = GemAbstract(GemJoinAbstract(PsmAbstract(_psm).gemJoin()).gem());
+        psm = PsmAbstract(_psm);
+        gem = _gem;
+        dai = DaiAbstract(PsmAbstract(_psm).dai());
+
+        uint256 gemDecimals = _gem.decimals();
+        uint256 daiDecimals = dai.decimals();
+        require(gemDecimals <= daiDecimals, "RwaOutputConduit3/invalid-gem-decimals");
+        to18ConvertionFactor = 10**(daiDecimals - gemDecimals);
```

Comment:
I think you can use `10 ** sub(dai.decimals(), _gem.decimals())` here and remove the 3 lines above

Reply:
I'd even use just `18`  instead of `dai.decimals()`

Reply:
done